lib/order.rb

```diff
@@ -0,0 +1,9 @@
+class Order
+
+  def initialize(customer, takeaway)
```

Comment:
Unused method argument - `customer`. If it's necessary, use `_` or `_customer` as an argument name to indicate that it won't be used. You can also write as `initialize(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `takeaway`. If it's necessary, use `_` or `_takeaway` as an argument name to indicate that it won't be used. You can also write as `initialize(*)` if you want the method to accept any arguments but don't care about them.